stock_available_to_promise_release/tests/test_reservation.py

```diff
@@ -546,6 +547,10 @@ def test_defer_creation_backorder(self):
         pickings = self._create_picking_chain(self.wh, [(self.product1, 20)])
         self.assertEqual(len(pickings), 1, "expect only the last out->customer")
         cust_picking = pickings
+        move_state = cust_picking.mapped("move_lines").mapped("state")
+        if len(move_state) == 1 and move_state[0] == "waiting":
+            cust_picking.state = "waiting"
```

Comment:
This code seems wrong, why do you update the state of the picking, it probably hides issues?